lab-david/app/entry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+
+angular.module('beerApp', [])
+.component('cowsayContainer', {
+  template: require('./cowsay.html'),
+  controller: ['$log','$q', '$http', function($log, $q, $http) {
```

Comment:
While I know this is from Duncan having worked on your code in class, no need to include ``$q`` or ``$http`` because they're not being utilized.